uktrade/data-hub-frontend
# Cls2 550 completed tasks project investment screen

## Description of change

[JIRA ticket](https://uktrade.atlassian.net/browse/CLS2-550)

This change allows the DataHub users to be able to see the completed tasks on the investment project screen. It removes the `archived` filter from the API calls (returning both completed and not completed Tasks) and instead shows a `COMPLETED` tag both in the list and details. It also removes the `Mark as complete` button when needed and converts the back button to a link as per requested.

## Test instructions

- When viewing an Investment Project, add a task. The task should be marked as not archived (no tag)
- Add a second task for good measure
- Pick the first task, view its details page and mark as completed
- This will redirect you to the tasks list page. Two tasks should still be visible (as opposed to just one). One of the two tasks should have a completed tag
- View the details page for the completed task. The completed tag should be under its title and the `mark as complete` button should not be available

## Screenshots

<img width="752" alt="Screenshot 2023-11-23 at 14 58 48" src="https://github.com/uktrade/data-hub-frontend/assets/1381563/e5566ccc-3379-4427-b3f0-98af4baa2e03">
<img width="752" alt="Screenshot 2023-11-23 at 14 58 59" src="https://github.com/uktrade/data-hub-frontend/assets/1381563/f3f0839c-c5aa-407a-b10d-4e64712d1e9f">
<img width="752" alt="Screenshot 2023-11-23 at 14 59 06" src="https://github.com/uktrade/data-hub-frontend/assets/1381563/4aaa6177-c2f8-4c60-b175-4cc3fae58b68">


## Checklist
